DevVehicle35-Motors/App/BusInteraction.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DevVehicle35_Motors.Models;
+
+namespace DevVehicle35_Motors.App
+{
+    internal class BusInteraction
+    {
+        internal static void BuildBus()
+        {
+            Bus busan = new ();
+            if (ValidateInput("Wish you add Online Entertainment?"))
+            {
+                busan.AddEntertainment();
+            }
+
+            Console.Clear();
+            if (ValidateInput("Wish you add a bathroom in the bus?"))
+            {
+                busan.AddBathroom();
+            }
+
+            Console.Clear();
+            if (ValidateInput("Wish you add more seats?"))
+            {
+                Console.WriteLine("Enter the amount of seat you wish to add");
+                int seatAmount = int.Parse(Console.ReadLine());
+                busan.AddSeats(seatAmount);
+            }
+
+            Console.Clear();
+            Console.WriteLine(busan.GetDescription());
+            Console.WriteLine("\nCONGRATULATION YOU HAVE BUILD A BUS SUCESSFULLY");
+        }
+
+        private static bool ValidateInput(string message)
```

Comment:
I think the name is generic but implementation is not. I think you should either change the method's name or find a way that the method can check the input for others vehicles other than "bus". Maybe it could receive another string so it the title show something like building a {type of vehicle}.